src/dynamics/video_player/player/player.html

```diff
@@ -10,6 +10,9 @@
      ba-on:touchstart="{{user_activity(true)}}"
      ba-styles="{{widthHeightStyles}}"
 >
+    <meta itemprop="name" content="{{title || 'Video Plauploaddateyer'}}" />
```

Comment:
Looks like there's a typo here

Reply:
Yes Natasha thanks that mentioned it. This Jetbrains IDE make computer so slow, when try to find something inside file put searched work in cursor poistion